### DISCUSSION TITLE
Tweak invert class css ?

### DISCUSSION THREAD

**Discussion by tianfanghan** [marp-team/marp#282] [Q&A] [answered]
Hi yhatt. Sorry for bother you again. 
Is there any way to tweak css for example H1 color for invert class? 
Even in md file or in custom theme css file?

**Comment by yhatt** [comment, marked as answer]
Try this:

```html
<style>
section.invert h1 {
  color: red;
}
</style>
```

The selector `section.invert h1` is meaning to target `h1` element within `<section>` slide container that has `invert` class.

**Reply by tianfanghan** [reply]
wow, perfect ! it's very nice of you.

**Comment by tianfanghan** [comment]
But the code:

```
<style>
section.invert h1 {
  color: red;
}
</style>
```

can not change the html file background color determined by 

```
::backdrop {
  background-color: white;
}
```

can we invert the html background also?

**Reply by tianfanghan** [reply]
@yhatt

**Reply by yhatt** [reply]
`::backdrop` selector in Marp / Marpit theme will be treated as the global style in the bunch of slides. The target element of `::backdrop` has located to the outside of the slide elements that can change the class via `class` local directive, and cannot apply specific style by the class of the slide.

**Reply by tianfanghan** [reply]
Wow, It's beyond my knowledge. Thanks again.
